### PR TITLE
[doc] Add a example of Incremental Snapshot based DataStream

### PR DESCRIPTION
**Pull request by zhaomin1423**

### CODE REVIEW

File: docs/content/connectors/oracle-cdc.md [hunk 1]
```diff
@@ -469,7 +469,63 @@ The Oracle CDC source can't work in parallel reading, because there is only one
...
+import com.ververica.cdc.connectors.base.options.StartupOptions;
+import com.ververica.cdc.connectors.base.source.jdbc.JdbcIncrementalSource;
+import com.ververica.cdc.connectors.oracle.source.OracleSourceBuilder;
+import com.ververica.cdc.debezium.JsonDebeziumDeserializationSchema;
+import org.apache.flink.api.common.eventtime.WatermarkStrategy;
+import org.apache.flink.streaming.api.environment.StreamExecutionEnvironment;
+
+import java.util.Properties;
+
+public class PostgresParallelSourceExample {
```
Comment (gtk96) [inline docs/content/connectors/oracle-cdc.md:489]:
```suggestion
public class PostgresParallelSourceExample {
```
public class OracleParallelSourceExample {

File: docs/content/connectors/oracle-cdc.md [hunk 2]
```diff
@@ -469,7 +469,63 @@ The Oracle CDC source can't work in parallel reading, because there is only one
...
+
+        StreamExecutionEnvironment env = StreamExecutionEnvironment.getExecutionEnvironment();
+        // enable checkpoint
+        env.enableCheckpointing(3000L);
+        // set the source parallelism to 4
+        env.fromSource(
+                        oracleChangeEventSource,
+                        WatermarkStrategy.noWatermarks(),
+                        "OracleParallelSource")
+                .setParallelism(1)
```
Comment (gtk96) [inline docs/content/connectors/oracle-cdc.md:520]:
```suggestion
                .setParallelism(4)
```

Comment (zhaomin1423) [reply docs/content/connectors/oracle-cdc.md:520]: Thanks.

**Comment by gtk96** [comment]
Thanks @zhaomin1423 contribution ，LGTM

**Review by leonardBang — APPROVED** [review]
Thanks @zhaomin1423 for the contribution, thanks @gtk96 for the review. LGTM

**Pull request merged**